packages/build/src/js-transform.ts

```diff
@@ -168,7 +190,7 @@ export function jsTransform(js: string, options: JsTransformOptions): string {
   }
   if (options.compileToEs5) {
     doBabelTransform = true;
-    presets.push(babelPresetEs2015NoModules);
+    presets.push(customBabelPresetEs2015);
```

Comment:
`plugins.push(...`? Probably works either way, but seems more clear now that it's not technically a preset.

Reply:
AFAICT, this would change the order? https://babeljs.io/docs/plugins/#pluginpreset-ordering

Reply:
https://babeljs.io/docs/plugins/#creating-a-preset kinda makes it sound like this counts as a preset.

Reply:
I think a preset is just an array of plugins, and yes it does affect ordering (if you look above in the transform functions I mentioned how things are ordered). I guess this is fine, and it definitely won't change the ordering!